modules/statistics/statistics.ts

```diff
@@ -42,71 +57,82 @@ export default class Statistics {
      * @static
      * @type {boolean}
      */
-    static audioLevelsEnabled = false;
+    static audioLevelsEnabled: boolean = false;
 
     /**
      * The interval for audio levels stats collection.
      * @static
      * @type {number}
      */
-    static audioLevelsInterval = 200;
+    static audioLevelsInterval: number = 200;
 
     /**
      * The interval for peer connection stats collection.
      * @static
      * @type {number}
      */
-    static pcStatsInterval = 10000;
+    static pcStatsInterval: number = 10000;
+
+    /**
+     * The interval for long tasks stats collection.
+     * @static
+     * @type {number}
+     */
+    static longTasksStatsInterval: number = 0;
 
     /**
      * Flag indicating whether third party requests are disabled.
      * @static
      * @type {boolean}
      */
-    static disableThirdPartyRequests = false;
+    static disableThirdPartyRequests: boolean = false;
 
     /**
      * Analytics adapter for sending events.
      * @static
      * @type {Object}
      */
-    static analytics = analytics;
+    static analytics: {
+        sendEvent: (eventName: string | Record<string, any>, properties?: Record<string, any>) => void;
+    } = analytics;
 
     /**
      * Array holding local statistics collectors.
      * @static
      * @type {Array}
      */
-    static localStats = [];
+    static localStats: LocalStats[] = [];
 
     /**
      * Local JID constant.
      * @static
      * @type {string}
      */
-    static LOCAL_JID = require('../../service/statistics/constants').LOCAL_JID;
+    static LOCAL_JID: string = require('../../service/statistics/constants').LOCAL_JID;
```

Comment:
Since `LOCAL_JID` is a named export, you can directly import it above with the rest of the imports and then initialized here.